cmd/post.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"path"
+
+	"github.com/spf13/cobra"
+)
+
+type Payload struct {
+	Content string `json:"content"`
```

Comment:
What does  `json:"xxx"` stand for?

Reply:
In JSON, it enables us to control key names?
https://www.digitalocean.com/community/tutorials/how-to-use-struct-tags-in-go

Reply:
> In JSON, it enables us to control key names? https://www.digitalocean.com/community/tutorials/how-to-use-struct-tags-in-go

Yes.
Struct tags is used in json, xml, db and more!
https://pkg.go.dev/encoding/json#Marshal
https://pkg.go.dev/encoding/xml#Marshal
https://pkg.go.dev/gopkg.in/yaml.v2#Marshal
https://github.com/jmoiron/sqlx

https://github.com/golang/go/wiki/Well-known-struct-tags